internal/terraform/workspace/workspace_manager.go

```diff
@@ -0,0 +1,161 @@
+package workspace
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-ls/internal/terraform/discovery"
+	"github.com/hashicorp/terraform-ls/internal/terraform/exec"
+	"github.com/hashicorp/terraform-ls/internal/terraform/lang"
+	"github.com/hashicorp/terraform-ls/internal/terraform/schema"
+)
+
+type workspaceManager struct {
+	ctx           context.Context
```

Comment:
Shouldn't attach `context.Context` to structs

Reply:
Fair - I think I can just create a fresh context for new workspaces instead.